### PR TITLE
LPS-105380 Move package

### PR DESCRIPTION
**Pull request by ling-alan-huang**
Hi Hugo,

Issue: https://github.com/liferay/liferay-portal/commit/bf30aaba9f9b9ff3a0243d5a211b32b583c82c11

### CODE REVIEW

**Comment by ling-alan-huang** [comment]
ci:test:sf

**Comment by ling-alan-huang** [comment]
ci:test:relevant

**Comment by ling-alan-huang** [comment]
ci:test:sf

File: modules/apps/archived/weather-web/src/main/resources/META-INF/resources/view.jsp [hunk 1]
```diff
@@ -65,7 +65,13 @@
 
 		<br />
 
-		<liferay-ui:message key="powered-by" /> <a href="http://www.openweathermap.org" target="_blank">Open Weather Map</a>
```
Comment (ling-alan-huang) [inline modules/apps/archived/weather-web/src/main/resources/META-INF/resources/view.jsp:68]: @alee8888 can I make this change? refer: https://github.com/liferay/liferay-portal/commit/8446a23180b029737c93d20c1b75345694d8b18a

**Comment by hhuijser** [comment]
ci:forward

**Comment by ling-alan-huang** [comment]
ci:forward

**Pull request closed without merging**